.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: node_js
+addons:
+  firefox: latest-esr
+script: npm run test
```

Comment:
https://github.com/nextcloud/vue-components/blob/master/.travis.yml
Please also add a compile check and a lint check :)